docs/docs/classic-ml/traditional-ml/sklearn/quickstart/quickstart-sklearn.ipynb

```diff
@@ -0,0 +1,1965 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "68c6fd65-c340-4f90-84de-b238a417604e",
+     "showTitle": false,
+     "tableResultSettingsMap": {},
+     "title": ""
+    },
+    "id": "cUsmLBhM8m-Y"
+   },
+   "source": [
+    "# Get Started with MLflow + Scikit-learn\n",
+    "\n",
+    "In this guide, we will show you how to train a model with scikit-learn and log your training using MLflow.\n",
+    "\n",
+    "We will be using the [Databricks Free Trial](https://mlflow.org/docs/latest/ml/getting-started/databricks-trial.html), which has built-in support for MLflow. The Databricks Free Trial provides an opportunity to use Databricks platform for free. If you haven't already, please register for an account via [this link](http://signup.databricks.com).\n",
```

Comment:
This link should be: 

`https://signup.databricks.com/?destination_url=/ml/experiments-signup?source=TRY_MLFLOW&dbx_source=TRY_MLFLOW&signup_experience_step=EXPRESS`